api/v1/model_types.go

```diff
@@ -27,3 +27,21 @@ type MetricResult struct {
 	Metrics         []MetricPoint `json:"metrics"`
 	LatestTimestamp time.Time     `json:"latestTimestamp"`
 }
+
+type Stats struct {
+	Average    uint64 `json:"average"`
+	Percentile uint64 `json:"percentile"`
```

Comment:
It is the 95th percentile. The underlying Statstore only provides one hardcoded percentile value as of now, the 95th percentile, due to a pending integration with CAdvisor's percentile derivations.

I can rename the struct field to NinetyFifth and consider reworking the StatStore implementation